certs/CMakeLists.txt

```diff
@@ -0,0 +1,7 @@
+#Copyright (c) Microsoft. All rights reserved.
```

Comment:
Hey Mollie, what is the technical need for this? Can't the executables just continue including the source files directly?

Reply:
You are correct, this is not required and does not change things technically  The reason I added this is because I think it makes the Sample CMakes easier to follow without the cert if-statement in the middle.  I also prefer the design of having a CMake for each directory, because I also think its easier to follow.  Putting it forward as an idea/proposal.